src/scripts/browserDeviceProfile.js

```diff
@@ -668,17 +668,43 @@ import browser from './browser';
         }
 
         if (canPlayVp8) {
+            // TODO: Remove vpx entry once servers are migrated
             profile.TranscodingProfiles.push({
                 Container: 'webm',
                 Type: 'Video',
-                AudioCodec: 'vorbis',
+                AudioCodec: webmAudioCodecs.includes('opus') ? 'opus' : 'vorbis',
                 VideoCodec: 'vpx',
```

Comment:
Maybe it would make sense to use `webmVideoCodecs` here and just append `vpx`. I'm not sure if av1 is currently supported when transcoding to webm though. It looks like it can be included in the mp4 transcoding profile now.

Reply:
- Just for clarity, we're trying to deprecate `vpx` here. Adding it to `webmVideoCodecs` is fine until jellyfin/jellyfin#6456 lands and is sufficiently migrated to.
- The `VideoCodec` field of a `TranscodingProfile` is expected to only have one codec as of now. So regardless, we'd have to add them all separately.
- av1 is transcoding is not fully supported by the server yet, though I might create a pull request for it. I don't imagine it will be heavily used since most servers won't be powerful enough to transcode in realtime.

Reply:
> The `VideoCodec` field of a `TranscodingProfile` is expected to only have one codec as of now. So regardless, we'd have to add them all separately.

Is this just a limitation of the code for the webm container, because every other video container contains multiple comma separated values for the `VideoCodec` field?

Reply:
You're right, I misremembered and had to double check on the server side. I've made the change now in a new commit.